src/components/Dropdown/Dropdown.st.css

```diff
@@ -137,3 +137,9 @@
 .root:alignment(center) DropdownOption {
     justify-content: center;
 }
+
+.divider {
+    margin: 10px 13px !important;
+    width: auto !important;
+    opacity: 0.2 !important;
+}
```

Comment:
Why do you need important?
Are you overriding the wix-ui-core styling?

Reply:
That's right :( 
I'll see if there's a way to remove them

Reply:
fixed